docs/api-reference/effects/overview.md

```diff
@@ -0,0 +1,7 @@
+# Overview
+
+The `@luma.gl/effects` module offers modular, reusable effects.
```

Comment:
Call out that these are screen space effects and give a few examples to make it clear what kinds of effects we're talking about.

Reply:
I think the intention is to add non-screenspace effects to this module later.
Maybe start a list of what's in this package here and link to sub-pages?